contrib/haproxy/template.py

```diff
@@ -0,0 +1,58 @@
+pkgname = "haproxy"
+pkgver = "2.9.3"
+pkgrel = 0
+build_style = "makefile"
```

Comment:
it ignores cflags/ldflags, you'd probably want to change

```
CFLAGS = ...
```

to

```
CFLAGS := ... $(CFLAGS)
```

same for ldflags

Reply:
isn't that what `make_use_env = True` is for ?

Reply:
make_use_env doesn't make makefiles that ignore cflags not ignore them, it replaces

````
make CFLAGS="default cflags" ...
```

with

```
CFLAGS="default cflags" ... make
```

which as the name implies uses the env instead of passing args directly. in this case, though, it seems all of the makefile LDFLAGS/CFLAGS are useless, so you can fix this by removing make_use_env

it will then fail because that passes LD=ld.lld and the makefile does $(LD) -o.. (links with LD instead of CC), which you can fix by changing the linker invocation to CC instead

Reply:
oh my bad I got confused, I will quickly patch it

Reply:
or well, it does complain if you do that, so patch the cflags instead :)

Reply:
it also needs !int and doesn't work with cfi